compiler_opt/rl/problem_configuration.py

```diff
@@ -95,6 +95,9 @@ def get_preprocessing_layer_creator(
   def get_runner(self, *args, **kwargs) -> compilation_runner.CompilationRunner:
     raise NotImplementedError
 
+  def get_nonnormalized_features(self) -> Iterable[str]:
```

Comment:
nit: move it after get_preprocessing_layer_creator and before runner to be consistent of other files